runner.go

```diff
@@ -140,10 +140,21 @@ func (r *Runner) Start() {
 
 	for {
 		// Enable quiescence for all templates if we have specified wait intervals.
-		if r.config.Wait.Min != 0 && r.config.Wait.Max != 0 {
-			for _, t := range r.templates {
+		for _, t := range r.templates {
+			for _, ctemplate := range r.configTemplatesFor(t) {
+				if ctemplate.Wait.Min != 0 && ctemplate.Wait.Max != 0 {
```

Comment:
What do you think about inverting the boolean statement and use `continue` if `Wait.Min` or `Wait.Max` is `0`, to reduce the indentation level.